tests/activation/test_prompts.py

```diff
@@ -0,0 +1,280 @@
+"""test that prompt behavior is correct in supported shells"""
+from __future__ import absolute_import, unicode_literals
+
+import os
+import subprocess
+import sys
+from collections import namedtuple
+from textwrap import dedent
+
+import pytest
+
+import virtualenv
+
+try:
+    from pathlib import Path
```

Comment:
always use pathlib2 👍

Reply:
:+1: I've been Python 3-only for all of my side projects for ~4yrs now, so I'm totally rusty on 2/3 compat!